### PR TITLE
Shorter syntax for the debug shell

### PR DESCRIPTION
**Pull request by Markcial**
It adds another dependency based on the library SuperClosure, but it leaves a shell call with a shorter syntax. Feel free to modify the function , reject the PR or whatever. I can have this only on my fork, but i thought it could be interesting for you

### CODE REVIEW

**Comment by bobthecow** [comment]
I've considered doing things with SuperClosure in the past, but have run into issues where PHP was too dumb about things to let me use it, so I'm not sure I'm comfortable 1) making it a dependency, and 2) recommending it over the `eval` version that works, for what is a relatively small gain :)

```php
eval(\Psy\sh());
\Psy\dbg();
```

**Comment by Markcial** [comment]
its a good point, after 3 hours of looking at the debug_backtrace I had suddenly felt really sad with the PHP functionalities, but i came into this, as a informational note the PR is here, but if you reject it won't be a shame and I will completely understand.

**Comment by bobthecow** [comment]
Now, if you wanted to write a PHP extension that added a `debug` keyword, *that* would be a huge win :)

**Comment by Markcial** [comment]
wish i could, i am not as badass writing extensions as nikic. Maybe in the long run. :)

**Comment by bobthecow** [comment]
Heh. Me too :)

I'm marking this as an enhancement and closing it for now.

**Comment by bobthecow** [comment]
This might actually fix the issues I was running into before: https://github.com/jeremeamia/super_closure/issues/34

… but it's hard to say without extensive testing :)

File: composer.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
         "symfony/console": "~2.3.10|~2.4.2|~2.5",
         "nikic/php-parser": "~1.0",
         "dnoegel/php-xdg-base-dir": "0.1",
-        "jakub-onderka/php-console-highlighter": "0.3.*"
+        "jakub-onderka/php-console-highlighter": "0.3.*",
+        "jeremeamia/SuperClosure": "@stable"
```
Comment (GrahamCampbell) [inline composer.json]: This is a super terrible version constraint, and is already broken....

Comment (bobthecow) [inline composer.json]: agreed. but we're likely not using it at all, so i didn't worry too much about it :)

**Comment by Markcial** [comment]
Good news everyone, i found a library for php extensions, is possible to have a shorter syntax without troubles only installing the module, some screenshots will follow
captura de pantalla 2015-01-29 a las 23 48 08
captura de pantalla 2015-01-29 a las 23 48 53

The library used is [PHP-CPP](http://www.php-cpp.com/) and it could be easily compiled, but it is a whole new level of dependencies, if you are interested i can open a whole new repository for the module and maybe some helper functions for psysh.

**Pull request closed without merging**